embedded-uart-common/test-config/sensirion_test_setup.cpp

```diff
@@ -0,0 +1,5 @@
+#include "CppUTest/CommandLineTestRunner.h"
```

Comment:
We're starting the duplication process, so I think it's time to include `embedded-common` here too as a submodule. To keep the changes minimal, I'd only import files from there but not move `embedded-uart-common` over just yet until we have a second uart driver, since we might want to change the folder structure which would affect all drivers.

Reply:
But the content of embedded-uart-common is totally different form embedded-common. even some of the test files have small changes.

Reply:
Yes, I was only concerned about the test files, but I see, this will also need generalization - apologies, at first it looked like a duplicate, so let's postpone.